Contest_Control_System_Requirements.md

```diff
@@ -841,12 +841,15 @@ according to the following:
         all.
     3.  For the remaining teams, the rank is determined by sorting the teams by 
         number of problems solved (descending).
-8.  The *award* of a team is:
-    1.  *gold* if rank is 1 through 4.
-    2.  *silver* if rank is 5 through 8.
-    3.  *bronze* if rank is 9 through 12+B.
-    4.  *ranked* if rank is not 1 through 12+B but the team is ranked.
-    5.  *honorable* otherwise.
+8.  The *awards* of a team are:
+    1.  `gold-medal` if rank is 1 through 4.
+    2.  `silver-medal` if rank is 5 through 8.
+    3.  `bronze-medal` if rank is 9 through 12+B.
```

Comment:
Did he (Bill) or did he not do away with the concept of "B" ?  Can we remove it?

Reply:
That was my understanding as well. (Both, that Bill has removed "B", and that @clevengr said to keep it for now).